tests/FakeItEasy.Analyzer.Tests/NonVirtualSetupAnalyzerTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace FakeItEasy.Analyzer.Tests
+{
+    using Helpers;

```

Comment:
An additional nitpick, which I'm not sure our StyleCop catches, but the preference is to use the full namespace in usings. Again, don't go out of your way to make a change, but if you happen to be in the area later…